client/app/dedicatedCloud/datacenter/drp/steps/common/first/dedicatedCloud-datacenter-drp-firstStep.html

```diff
@@ -0,0 +1,59 @@
+<div class="text-center"
+     data-ng-if="$ctrl.loading">
+    <oui-spinner></oui-spinner>
+</div>
+
+<div data-ng-if="!$ctrl.loading">
+    <p data-translate="dedicatedCloud_datacenter_pcc_name"
+       data-translate-values="{ 'pcc': $ctrl.drpInformations.primaryPcc.description }"></p>
+    <form name="$ctrl.primaryPcc"
+          novalidate>
+        <oui-field data-label="{{:: 'dedicatedCloud_datacenter_main_datacenter_name' | translate }}"
+                   data-size="l">
+            <oui-select name="datacenterId"
+                        data-match="formattedName"
+                        data-model="$ctrl.drpInformations.primaryDatacenter"
+                        data-items="$ctrl.availableDatacenters"
+                        data-placeholder="{{:: 'dedicatedCloud_datacenter_main_datacenter_name_placeholder' | translate }}"
+                        data-disabled="!$ctrl.drpInformations.hasDatacenterWithoutHosts">
+            </oui-select>
+        </oui-field>
+
+        <div data-ng-if="$ctrl.availableIpAddress.length === 0">
+            <p class="oui-label"
+               data-translate="dedicatedCloud_datacenter_main_ip_address_pool_available"></p>
+            <span data-translate="dedicatedCloud_datacenter_order_ip"></span>
+            <button class="oui-button oui-button_link px-0 py-0"
+                    type="button"
+                    data-ng-click="$ctrl.openModalOrderIpBlock()">
+                <span data-translate="global_feedback_contact_link"></span>
```

Comment:
`data-translate` directive could be directly applied to the parent element.

Couldn't be turned into a regular `<oui-button>` instead?

Reply:
I didn't used a `oui-button` to have the correct button placement, what i can't do with a `oui-button`